### ISSUE TITLE
[🐞] qwik-auth error

### ISSUE THREAD

**Issue by ulic75** [BuilderIO/qwik#2973]
### Which component is affected?

Qwik Runtime

### Describe the bug

When I try to impliment qwik-auth I receive this error
`Cannot read properties of undefined (reading 'serverAuthQrl')`

`plugin@auth.ts`
```js
import type { Provider } from '@auth/core/providers';
import GitHub from '@auth/core/providers/github';
import Auth0 from '@auth/core/providers/auth0';
import { serverAuth$ } from '@builder.io/qwik-auth';

export const { onRequest, useAuthSession, useAuthSignup, useAuthLogout } = serverAuth$(
  ({ env }) => ({
    secret: env.get('AUTH_SECRET'),
    trustHost: true,
    providers: [
      GitHub({
        clientId: env.get('GITHUB_ID')!,
        clientSecret: env.get('GITHUB_SECRET')!,
      }),
      Auth0({
        clientId: env.get('AUTH0_ID') || '',
        clientSecret: env.get('AUTH0_SECRET') || '',
        issuer: env.get('AUTH0_ISSUER'),
      }),
    ] as Provider[],
  })
);
```

### Reproduction

https://stackblitz.com/edit/qwik-starter-abhsz2?file=src/routes/plugin@auth.ts

### System Info

```shell
Stackblitz
```

**Comment by mmarkelov** [comment]
Not sure, but guess that it is not imported correctly `Cannot read properties of undefined (reading 'default')`

**Comment by manucorporat** [comment]
Should be fixed! in 0.0.2!
